versioned_docs/version-0.3.2/index.md

```diff
@@ -45,14 +44,13 @@ Noir can be used for a variety of purposes.
 ### Ethereum Developers
 
 Noir currently includes a command to publish a contract which verifies your Noir program. This will
-be modularised in the future, however as of the alpha you can use the `contract` command to create
-it.
+be modularised in the future; however, as of the alpha, you can use the `contract` command to create
```

Comment:
I actually think this is out of date and should be `codegen-verifier` but I'll let the devrel team confirm

Reply:
Yeah it's `codegen-verifier`, can you please correct @brunny-eth ? I don't have access to your fork